.gitignore

```diff
@@ -14,6 +14,7 @@ dist/
 downloads/
 eggs/
 .eggs/
+pyenv.cfg
```

Comment:
great. I can still see the pyvenv.cfg here. Please git remove the file itself too

Reply:
Done